mail-mta/protonmail-bridge-bin/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you should add gentoo proxy maintainers last, as you are not a gentoo developer.

Reply:
I don't mind maintaining it. You can add me as a dev.

Reply:
https://www.gentoo.org/get-involved/become-developer/

Reply:
@Zlogene added proxy-maint